src/mono/wasm/runtime/driver.c

```diff
@@ -25,6 +25,12 @@
 
 #include <mono/metadata/mono-private-unstable.h>
 
+#ifndef DISABLE_THREADS
+
+#include <emscripten/threading.h>
+#include <emscripten/wasm_worker.h>
```

Comment:
We shouldn't be using anything from wasm_worker.h - that's the non-pthreads alternate worker interface that they're developing. It's a completely separate set of workers.

Reply:
Yeah the docs are really bad so I tried that header first, I think it's from threading.h.